src/main/scala/nl/knaw/dans/lib/logging/servlet/ResponseLogFormatter.scala

```diff
@@ -78,6 +81,9 @@ trait ResponseLogFormatter {
    * @return the formatted actionHeaders
    */
   protected def formatActionHeaders(actionHeaders: ActionHeadersMap): ActionHeadersMap = {
+    // Review comment: so far no headers encountered that contained sensitive data
+    // don't now if any common header may, or we will sooner or later might develop
+    // our own custom header with sensitive data, better safe than sorry with this hook
```

Comment:
good point; we'll keep it even if we don't have overrides yet.
I have another idea on this, but I'll have to work that out some more. I'll do a separate PR on that later for you to review.

Reply:
@jo-pol see https://github.com/rvanheest-DANS-KNAW/dans-scala-lib/pull/2